### PR TITLE
fix(components): [select-v2] emit('blur') does not carry parameter

### PR DESCRIPTION
**Pull request by sechi747**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

[相关issue #9523 ](https://github.com/element-plus/element-plus/issues/9523)
根据官方文档，blur事件的回调应携带 Event 参数
image

### CODE REVIEW

File: packages/components/select-v2/src/useSelect.ts [hunk 1]
```diff
@@ -468,7 +468,7 @@ const useSelect = (props: ExtractPropTypes<typeof SelectProps>, emit) => {
     }
   }
 
-  const handleBlur = () => {
+  const handleBlur = (event: Event) => {
```
Comment (chenxch) [inline packages/components/select-v2/src/useSelect.ts:471]: Maybe this should be `Focusevent`.

Comment (sechi747) [reply packages/components/select-v2/src/useSelect.ts]: Yes, I have fixed it. Please review again.

**Comment by chenxch** [comment]
Need to pull dev.

**Comment by sechi747** [comment]
> Need to pull dev.

done.

**Review by chenxch — APPROVED** [review]
LGTM.

**Review by xiaoxian521 — APPROVED** [review]

**Pull request merged**